internal/reflect/into.go

```diff
@@ -109,11 +111,19 @@ func BuildValue(ctx context.Context, typ attr.Type, val tftypes.Value, target re
 		// all that's left to us now is to set it as an empty value or
 		// throw an error, depending on what's in opts
 		if !opts.UnhandledUnknownAsEmpty {
-			err := fmt.Errorf("unhandled unknown value")
+			typTypeStr := reflect.TypeOf(typ).String()
```

Comment:
Is this necessary for anonymous structs types? Given an `attr.Type`, I think we can just call `typ.String()` and anticipate folks implementing their types correctly, rather than potentially returning some confusing type details.

Reply:
I've removed this in both `unknown` and `null` cases and have switched to using `typ.String()` as you suggested.